src/i2c/master.rs

```diff
@@ -31,6 +31,22 @@ pub enum Speed {
     High,
 }
 
+/// Master SCL Low time. Specifies the minimum low time in clock cycles that will be asserted by this master on SCL.
+#[cfg(feature = "mimxrt685s")]
```

Comment:
why only 685?

Reply:
I'm not sure I understand the question - 633 has a similar thing for its enum type on line 44 if that's what you're asking?

Reply:
oh, cool, thanks! that seems way cleaner, will do that instead